site-src/implementations/gateways.md

```diff
@@ -93,3 +95,8 @@ Issue](https://github.com/istio/istio/issues/55768).
 gateway that can run [independently](https://gateway-api-inference-extension.sigs.k8s.io/guides/#__tabbed_3_3), as an [Istio waypoint](https://kgateway.dev/blog/extend-istio-ambient-kgateway-waypoint/),
 or within your [llm-d infrastructure](https://github.com/llm-d-incubation/llm-d-infra) to improve accelerator (GPU)
 utilization for AI inference workloads.
+
+## Kubvernor Rust API Gateway
```

Comment:
Similar to https://github.com/kubernetes-sigs/gateway-api-inference-extension/pull/1313/files#r2316579012 (the description can state that it's a Rust-based API gateway).